atk-sys/Cargo.toml

```diff
@@ -34,5 +34,5 @@ name = "atk-sys"
 repository = "https://github.com/gtk-rs/sys"
 version = "0.9.0"
 
-["package.metadata.docs.rs"]
-features = ["dox", "v2_32"]
+[package.metadata.docs.rs]
```

Comment:
Strange, but my regen don't have empty line before this.
Any idea why?

Reply:
Unfortunately no. I had to make the changes by hand. I think it's a toml bug but not sure. We should check if updating the toml version fixes this bug. Don't have time now but will definitely look into it later.